R/utilities.R

```diff
@@ -821,42 +821,40 @@ list_dataset_columns <- function(dataset, year = 2021, analytic_only = F){
   # create a negate function of %in% for readability
   '%!in%' = Negate('%in%')
 
-  if(dataset %!in% list_apde_data()) {
-    dat = match.arg(dataset, list_apde_data())
-    warning(paste0('non-exact database chosent. Attempting best match using "', dat,'" instead of "', dataset, '".'))
-  }
+  dataset = match.arg(dataset, c('birth', 'hys'))
```

Comment:
If keeping match.arg(), my thinking is the user should receive feedback about matches made if made. Why remove the warning?

Reply:
In the most recent commit, I've combined the warning and error into one. Now if the user asks for a dataset that is not implemented, it'll error out and tell them which ones are implemented.

Reply:
then, should this match.arg operation be removed?

Reply:
Good point. The most recent commit has it removed and instead I add a stopifnot to prevent dataset from being not length 1